examples/server/tests/utils.py

```diff
@@ -26,7 +26,7 @@
 import wget
 
 
-DEFAULT_HTTP_TIMEOUT = 10 if "LLAMA_SANITIZE" not in os.environ else 30
+DEFAULT_HTTP_TIMEOUT = 12 if "LLAMA_SANITIZE" not in os.environ else 30
```

Comment:
You can also modify the timeout per-request, for example:

`server.make_request(..., timeout=12)`

Reply:
Done in https://github.com/ggerganov/llama.cpp/pull/9639/commits/2d51c459c6fcc76f320abd54d4d91f8bb2120f6b